compiler warnings, cc (gcc) 8.1.1 and clang 6.0.1

git master

cc warning
```
[ 59%] Building C object CMakeFiles/pulse.dir/src/apulse-stream.c.o
/root/apulse/src/apulse-stream.c: In function ‘data_available_for_stream’:
/root/apulse/src/apulse-stream.c:93:13: warning: enumeration value ‘SND_PCM_STATE_PRIVATE1’ not handled in switch [-Wswitch]
             switch (snd_pcm_state(s->ph)) {
             ^~~~~~
```

clang warning
```
[ 59%] Building C object CMakeFiles/pulse.dir/src/apulse-stream.c.o
/root/apulse/src/apulse-stream.c:93:21: warning: enumeration value 'SND_PCM_STATE_PRIVATE1' not handled in switch [-Wswitch]
            switch (snd_pcm_state(s->ph)) {
                    ^
1 warning generated.
```

--- a/src/apulse-stream.c
+++ b/src/apulse-stream.c
@@ -147,6 +147,9 @@ data_available_for_stream(pa_mainloop_api *a, pa_io_event *ioe, int fd,
                     "SND_PCM_STATE_DISCONNECTED state. Giving up.",
                     s->name ? s->name : "", s->c->name ? s->c->name : "");
                 break;
+            default:
+                // avoid compiler warnings of unhandled (library-private) enum values
+                break;
             }
 
 #if HAVE_SND_PCM_AVAIL

